examples/index.js

```diff
@@ -19,13 +19,23 @@ $(() => {
   editable.add('.formatting-example p')
   setupTooltip()
 
+  editable.add('.styling-example p')
+
   var secondExample = document.querySelector('.formatting-example p')
   updateCode(secondExample)
 
   editable.on('change', (elem) => {
     if (elem === secondExample) updateCode(elem)
   })
 
+  // Styling
+  // -------
+
+  $('select[name="editable-styles"]').on('change', function() {
+    $('.styling-example p').removeClass('example-style-default example-style-dark')
+                           .addClass('example-style-' + $(this).val())
```

Comment:
I wanted to keep it explicit, hence the two CSS styles `default` and `dark`. We could simplify the code, remove the empty default styling, and call [`toggleClass()`](https://api.jquery.com/toggleClass/). Whichever works for you, @peyerluk.